Libraries/Text/TextInput/RCTBackedTextInputDelegate.h

```diff
@@ -45,6 +45,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)textInputDidCancel;  // Handle `Escape` key press.
 #endif // ]TODO(macOS GH#774)
+- (BOOL)textInputShouldHandlePaste:(id<RCTBackedTextInputViewProtocol>)sender; // Return `YES` to have the paste event handled normally. Return `NO` to disallow it and handle it yourself. // TODO(macOS GH#774)
```

Comment:
Should it be inside the ifdef above?

Reply:
We *can* when we are also changing adding #ifdef to these lines
```
- (void)paste:(id)sender
{
  if ([self.textInputDelegate textInputShouldHandlePaste:self]) {
    [super paste:sender];
    _textWasPasted = YES;
  }
}
```
in RCTUITextView.m